tests/jest/config/webpack/loaders/devSitePackage.test.jsx

```diff
@@ -0,0 +1,58 @@
+import 'regenerator-runtime/runtime';
+
+const path = require('path');
+const { runLoaders } = require('loader-runner');
+
+describe('devSitePackage loader test', () => {
+  it('when both repository url and repository directory are specified', (done) => {
+    runLoaders({
+      resource: path.resolve(__dirname, './pretendPackageOne.json'),
+      loaders: [
+        path.resolve(__dirname, '../../../../../config/webpack/loaders/devSitePackage'),
+      ],
+    }, (err, result) => {
+      if (err) return done(err);
+      expect(result.result[0]).toMatchSnapshot();
+      return done();
+    });
+  });
+
+  it('when only repository url is specified', (done) => {
```

Comment:
you could also add the test for when repository url is specified and it is not a git repo. so that we cover all the scenarios.

Reply:
Also test a git repo that doesn't begin with git+
`"url": "https://github.com/cerner/terra-dev-site.git"`

Reply:
Added the requested two tests here : https://github.com/cerner/terra-dev-site/pull/243/commits/3acf9f958a72e6ad031400f2c566813e15a3c78d